tests/test_datadog.py

```diff
@@ -100,10 +100,12 @@ def test_metrics(method_name):
     mocked_method.assert_called_with(
         metric_prefix + '.' + metric_name,
         value=value,
-        tags=[
-            key + ':' + value
-            for key, value in tags.items()
-        ]
+        tags=sorted(
+            [
+                key + ':' + value
```

Comment:
For consistency with the way it's handled above, it might be worth switching to `.format()` here...but that's just a thought.